### DISCUSSION TITLE
GetBlock fix for parameter maxSupportedTransactionVersion not updated in Nuget packages

### DISCUSSION THREAD

**Discussion by lowki1119** [bmresearch/Solnet#469] [General]
Hello guys, is this change supposed to be available in NugGet Packages ? Thanks

**Comment by BifrostTitan** [comment]
## Latest Nuget Packages
- [Solnet.Rpc](https://www.nuget.org/packages/Solana.Rpc/)
- [Solnet.Wallet](https://www.nuget.org/packages/Solana.Wallet/)
- [Solnet.Programs](https://www.nuget.org/packages/Solana.Programs/)
- [Solnet.Extensions](https://www.nuget.org/packages/Solana.Extensions/)
- [Solnet.Keystore](https://www.nuget.org/packages/Solana.Keystore/)
